src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -18,7 +18,11 @@ export default async function Home() {
 
   return (
     <div data-partial="landing">
-      <script type="module" src="/nextjs_migration/client/js/transitionObserver.js" async></script>
+      <script
+        type="module"
+        src="/nextjs_migration/client/js/transitionObserver.js"
+        async
+      ></script>
```

Comment:
Personal curiosity, why use `<script>` here versus Next's `<Script>` like we do below on L26?
Or is it due to the `async` prop? And if so, could we use `<Script strategy="lazyOnload".../>` to do the same thing? https://nextjs.org/docs/app/api-reference/components/script#strategy

Reply:
This particular `<script>` tag was converted back to `<Script>`. With our current setup for the in-between state in which we mostly ported the site to be served by Next.js instead of Express as a first step. Otherwise, some of the scripts that manipulate DOM in a non-React way are executing a bit late — only after hydration. More context here: https://github.com/mozilla/blurts-server/pull/3137.